src/Microsoft.ML.Transforms/Text/WordTokenizeTransform.cs

```diff
@@ -151,249 +109,354 @@ private static VersionInfo GetVersionInfo()
                 loaderAssemblyName: typeof(DelimitedTokenizeTransform).Assembly.FullName);
         }
 
-        public override bool CanSavePfa => true;
-
-        private readonly ColInfoEx[] _exes;
-
-        // Cached type of the output column(s).
-        private readonly ColumnType _columnType;
-
         private const string RegistrationName = "DelimitedTokenize";
 
-        /// <summary>
-        /// Public constructor corresponding to SignatureDataTransform.
-        /// </summary>
-        public DelimitedTokenizeTransform(IHostEnvironment env, Arguments args, IDataView input)
-            : base(env, RegistrationName, Contracts.CheckRef(args, nameof(args)).Column,
-                input, TestIsTextItem)
+        public sealed class ColumnInfo
         {
-            // REVIEW: Need to decide whether to inject an NA token between slots in ReadOnlyMemory inputs.
-            Host.AssertNonEmpty(Infos);
-            Host.Assert(Infos.Length == Utils.Size(args.Column));
-
-            _exes = new ColInfoEx[Infos.Length];
-            for (int iinfo = 0; iinfo < Infos.Length; iinfo++)
-                _exes[iinfo] = new ColInfoEx(args, iinfo);
+            public readonly string Input;
+            public readonly string Output;
+            public readonly char[] Separators;
 
-            _columnType = new VectorType(TextType.Instance);
-            Metadata.Seal();
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="input">Name of input column.</param>
+            /// <param name="output">Name of output column.</param>
+            /// <param name="separators">Casing text using the rules of the invariant culture.</param>
+            public ColumnInfo(string input, string output, char[] separators)
```

Comment:
>separators [](start = 66, length = 10)

I wish i can set default value to new[] {' '}, but I can't. Part of me wants to check is it empty and set to new[]{' '}, but I'm not sure about it.